topic_state: #lidar参数
  - {
      sub_topic: "/calibration/sensor/rslidar/PointCloud2",
      pub_topic: "/cti/web/rslidar",
      name: "rslidar",
      rank: 50,
    }
  - {
      sub_topic: "/calibration/sensor/rslidar/Bpearl_PointCloud2",
      pub_topic: "/cti/web/rslidarr_Bpearl",
      name: "Bpearl",
      rank: 51,
    }
  - {
      sub_topic: "/calibration/sensor/lslidar/back/lh_lslidar_pointcloud",
      pub_topic: "/cti/web/rslidarr_lh_back",
      name: "lh_lslidar_pointcloud",
      rank: 52,
    }
  - {
      sub_topic: "/calibration/sensor/lhlidar/lh_middle_pointcloud2",
      pub_topic: "/cti/web/rslidar_mid_scan",
      name: "lh_middle",
      rank: 53,
    }
  - {
      sub_topic: "/calibration/livox/lidar",
      pub_topic: "/cti/web/rslidar_livox",
      name: "livox_lidar",
      rank: 54,
    }
  - {
      sub_topic: "/calibration/livox/lidar/front_left",
      pub_topic: "/cti/web/livox/lidar/front_left",
      name: "front_left",
      rank: 55,
    }
  - {
      sub_topic: "/calibration/livox/lidar/rear_right",
      pub_topic: "/cti/web/livox/lidar/rear_right",
      name: "rear_right",
      rank: 56,
    }
calibration_launch: #
  - {
      file: "calibration_v5.launch.py",
      vehicle_type: "null",
    }
  - {
      file: "calibration_v6.launch.py",
      vehicle_type: "null",
    }
  - {
      file: "calibration_v7.launch.py",
      vehicle_type: "null",
    }
  - {
      file: "calibration_v8.launch.py",
      vehicle_type: "v7.1",
    }
  - {
      file: "calibration_v8.launch.py",
      vehicle_type: "v7.2",
    }
  - {
      file: "calibration_v8.launch.py",
      vehicle_type: "v7.3",
    }

